### DISCUSSION TITLE
What's the point in this library if anything written to the console is sent to cloudwatch anyway?

### DISCUSSION THREAD

**Discussion by jgtestw** [aws/aws-logging-dotnet#282] [Q&A] [answered]
What's the point in using this library, when I can simply log to the console and then anything written to the console will get automatically sent to CloudWatch (not sure how that actually happens but it does).

**Comment by bhoradc** [comment, marked as answer]
Hi @jgtestw, 

The `aws-logging-dotnet` library provides a consistent approach to logging across various AWS services and environments. While some services like AWS Lambda automatically send logs to CloudWatch Logs, others like EC2, ECS and others do not have built-in support for sending logs to CloudWatch.

Therefore, by using this library, you can ensure that your application logs are consistently sent to CloudWatch Logs, regardless of the AWS service or environment in which the application is running.

Regards, 
Chaitanya
